hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/container/common/statemachine/commandhandler/TestBlockDeletion.java

```diff
@@ -392,16 +392,15 @@ public void testContainerStatisticsAfterDelete() throws Exception {
       });
     });
 
+    LogCapturer logCapturer = LogCapturer.captureLogs(ReplicationManager.LOG);
```

Comment:
This part is missing:

```
legacyEnabled ? LegacyReplicationManager.LOG  :
```

but looks like the test passes anyway.

Reply:
Ah, well spotted. I will fix that and squash the commit and force push the branch for another run.